src/gpodder/dbusproxy.py

```diff
@@ -36,27 +38,101 @@ def first_line(txt):
         return lines[0]
 
 
-class DBusPodcastsProxy(dbus.service.Object):
+def or_empty_str(txt):
+    """can't marshal the None value as a string, so use this wrapper to be safe"""
```

Comment:
This style of docstring triggers the flake8 linter (D400 and D403). Docstrings should be capitalized and end with a period.